lib/resources/oracledb_session.rb

```diff
@@ -56,11 +56,11 @@ def query(q)
         p = :parse_csv_result
       else
         bin = @sqlplus_bin
-        opts = "SET MARKUP HTML ON\nSET FEEDBACK OFF"
+        opts = "SET MARKUP HTML ON;\nSET FEEDBACK OFF;"
```

Comment:
Are these necessary? My understanding is `SET` commands are not SQL statements and therefore do not require the semicolons. In my limited testing, the semicolons are not needed here.